### ISSUE TITLE
Header: ShadowRoot

### ISSUE THREAD

**Issue by tfrijsewijk** [dso-toolkit/dso-toolkit#2365]

**Comment by hansgrimm** [comment]
@iterox Ik ben al een flinke tijd aan het speuren naar de reden waarom de styling van o.a. dropdown sjtuk is, en het valt me op dat de class `sc-dso-header` in de nieuwe situatie op diverse plekken ontbreekt. Ik zie dat daar ook styling aan wordt opgehangen. zou dat een oorzaak kunnen zijn?

FYI: ik ben behoorlijk afgedraaid; de hele nacht opgebleven voor de USA-verkieziengen, en nu gevloerd door het resultaat...

**Comment by tfrijsewijk** [comment]
De `sc-` prefixed classes is wat de Stencil Compiler doet als componenten met `scoped: true` worden gecompileerd. Stencil voegt deze prefix toe aan classes in stylesheets en het DOM.

Eric heeft de style encapsulation van `scoped` op `shadow` gezet. De prefix is dus verdwenen.

**Comment by hansgrimm** [comment]
Ja, zoiets vermoedde ik al. Zit er nu alleen op te studeren hoe dat precies stuk gegaan is, en hoe ik dat kan fixen zonder alles meteen overhoop te halen. Er zijn verschillende componenten bij betrokken.

**Comment by tfrijsewijk** [comment]
Met Dmitry heb ik afgesproken dat we het werk in deze branch rebasen op een recente versie en op NPM als feature release uitbrengen. Michiel schiet een story op hun backlog in waarmee Dmitry een impact analyze voor Header met shadow root doet. We verwachten dat er functioneel geen impact zou moeten zijn.

Dmitry heeft wel al aangegeven dat ze de tests moeten aanpassen omdat de elementen waar de Cypress tests van het Portaal mee draaien in een shadow root komen te staan.

@MichielDHVD Zou jij een ticket op jullie backlog willen prepareren? We geven een seintje als er een branch release is gedaan en er getest kan worden.
